test/tst.layout.js.out

```diff
@@ -668,16 +668,16 @@ summary:
      loadbalancer         -                2
      ops                  -                1
      madtom               -                1
-     reshard              -                 
-     pgstatsmon           -                 
-     garbage-collector     -                 
-     prometheus           -                 
+     reshard              -
+     pgstatsmon           -
+     garbage-collector     -
+     prometheus           -
```

Comment:
Ditto on tests and trailing whitespace. I haven't tried, so I'm not sure.

Reply:
I fixed this up and verified that the layout test passes now.